### DISCUSSION TITLE
GliNet AX1800的支持会更新吗？

### DISCUSSION THREAD

**Discussion by Jinnrry** [coolsnowwolf/lede#10411] [Q&A / 快速问答]
https://github.com/coolsnowwolf/openwrt-gl-ax1800

看到有这个库，但是这个库还在19版本，后续会继续同步更新吗？另外，ax1800为什么要单独开一个仓库，不合到一起呢？

**Comment by monw** [comment]
目前ipq6018还是qsdk的4.4.60内核比较稳定。
